app/models/comment.rb

```diff
@@ -0,0 +1,23 @@
+class Comment < ActiveRecord::Base
+
+  self.inheritance_column = :_type_disabled
+
+  belongs_to :assignment, foreign_key: :submission_id
+  belongs_to :exercise
+
+  validates_presence_of :exercise_id, :submission_id, :type, :content, :author, :date
+
+  def self.parse_json comment_json
+    comment = comment_json.delete('comment')
+    puts comment
+    comment['author'] = comment.delete('email')
+    comment_json.merge(comment)
+  end
+
+  def mark_as_read
```

Comment:
Done!

El lun., 4 de abr. de 2016 a la(s) 20:22, Franco Leonardo Bulgarelli <
notifications@github.com> escribió:

> In app/models/comment.rb
> <https://github.com/mumuki/mumuki-atheneum/pull/490#discussion_r58465549>:
>
> > +
> > +  self.inheritance_column = :_type_disabled
> > +
> > +  belongs_to :assignment, foreign_key: :submission_id
> > +  belongs_to :exercise
> > +
> > +  validates_presence_of :exercise_id, :submission_id, :type, :content, :author, :date
> > +
> > +  def self.parse_json comment_json
> > +    comment = comment_json.delete('comment')
> > +    puts comment
> > +    comment['author'] = comment.delete('email')
> > +    comment_json.merge(comment)
> > +  end
> > +
> > +  def mark_as_read
>
> I would rename this to just read!
>
> —
> You are receiving this because you authored the thread.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/mumuki/mumuki-atheneum/pull/490/files/90a3c24c8a49ec68453cd793dca6fa261d8c69f4#r58465549>
>